.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
```

Comment:
I think we don't need this file. Each dev uses a different configuration for the VSCode (or use another IDE)

Reply:
Fixed